.github/workflows/sanity_checks_utcs.py

```diff
@@ -0,0 +1,264 @@
+import json
+import os
+import re
+
+CORRECT_UNICODE = "\u2713"
+WRONG_UNICODE = "\u2715"
+CONNECTOR_CATEGORY = ['Analytics and SIEM', 'Authentication', 'Case Management', 'Threat Intelligence', 'Database',
+                      'Deception', 'Email Gateway', 'Endpoint Security', 'Identity and Access Management',
+                      'Authentication', 'Insider Threat', 'IT Services', 'Network Security', 'Utilities',
+                      'Vulnerability and Risk Management', 'OT & IoT Security', 'Communication and Coordination',
+                      'Compliance and Reporting', 'FortiSOAR Essentials', 'Monitoring',
+                      'Firewall and Network Protection', 'Threat Hunting and Search', 'Web Application',
+                      'Task Management', 'Enterprise mobility management', 'Digital assistant', 'Automation controller',
+                      'Logging', 'Query Service', 'Threat Detection', 'Breach and Attack Simulation (BAS)',
+                      'ML Service', 'Message Queueing Service', 'Security Posture Management', 'Compute Platform',
+                      'Storage', 'Asset Management', 'Container Services', 'IT Service Management',
+                      'Attack surface management', 'Malware Analysis', 'Cloud access security broker (CASB)',
+                      'Cloud Security', 'Email Server', 'Email Security',
+                      'DevOps and Digital Operations and Digital Operations', 'Ticket Management', 'Other']
+OPERATION_CATEGORY = ["investigation", "containment", "remediation", "miscellaneous"]
+PARAMETER_CATEGORY = ["text", "textarea", "integer", "datetime", "select", "multiselect", "checkbox", "password",
+                      "json", "apiOperation"]
+
+
+def get_info_file_path():
+    info_file_path = None
+    for dirname, dirnames, filenames in os.walk('.'):
+        if dirname in [".git", ".github"]:
+            continue
+        if "info.json" in filenames:
```

Comment:
These utility functions can be part of utility class e.g. utilities.py and can be referred here